grafana_objects/client_grafana_objects.go

```diff
@@ -0,0 +1,40 @@
+package grafana_objects
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/logzio/logzio_terraform_client/client"
+)
+
+const (
+	grafanaObjectsDashboardsByUID        = "%s/v1/grafana/api/dashboards/uid/%s"
+	grafanaObjectsDashboardsCreateUpdate = "%s/v1/grafana/api/dashboards/db"
```

Comment:
```suggestion
	grafanaObjectServiceEndpoint        = "%s/v1/grafana/api/dashboards"
```

Please add a `grafanaObjectServiceEndpoint` const that will have the base service endpoint.
Then, change the existing 2 consts to use it and move them to the file where it's used (grafanaObjectsDashboardsByUID to the get file, grafanaObjectsDashboardsCreateUpdate to the create update file).
You can see an example here:
https://github.com/logzio/logzio_terraform_client/blob/master/archive_logs/client_archive_logs.go#L8
and here:
https://github.com/logzio/logzio_terraform_client/blob/master/archive_logs/client_archive_logs_retrieve.go#L11

Reply:
done